lnwallet/channel.go

```diff
@@ -2476,6 +2476,7 @@ func (lc *LightningChannel) InitCooperativeClose() ([]byte, *chainhash.Hash, err
 
 	// As everything checks out, indicate in the channel status that a
 	// channel closure has been initiated.
+	lc.MarkChannelAsPendingClose(false)
```

Comment:
The channel shouldn't be marked as closed here as in this context, we don't know if the closure transaction has been broadcast or not.